api/tests/Api/ContentNodes/ChecklistNode/CreateChecklistNodeTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Tests\Api\ContentNodes\ChecklistNode;
+
+use App\Entity\ContentNode\ChecklistNode;
+use App\Tests\Api\ContentNodes\CreateContentNodeTestCase;
+
+/**
+ * @internal
+ */
+class CreateChecklistNodeTest extends CreateContentNodeTestCase {
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->endpoint = '/content_node/checklist_nodes';
+        $this->entityClass = ChecklistNode::class;
+        $this->defaultContentType = static::getFixture('contentTypeChecklist');
+    }
+
+    /**
+     * payload set up.
+     */
+    public function getExampleWritePayload($attributes = [], $except = []) {
+        return parent::getExampleWritePayload(
+            array_merge(
+                [
+                    'addChecklistItemIds' => null,
+                    'removeChecklistItemIds' => null,
```

Comment:
are these 2 attributes needed for the create request?

Reply:
Actually no. But `parent::getExampleWritePayload(..)` sets values by itself (because of the examples).

Reply:
So to send the requests as they would be sent by the frontend, you could add  these properties to the except array